.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-24.04]
-        bazel_version: [bazelbuild/8.1.0]
+        bazel_version: [bazelbuild/8.1.1]
```

Comment:
Won't we need CI to be on our custom version of Bazel, since that's the environment in which the bug is occurring?

Reply:
It should still use the version in Bazelisk. I believe this is just the version it sets up initially. Otherwise the CI builds wouldn't have passed when using the custom bazel version + multiplex sandboxing.

Reply:
But good point, I'll try to update this

Reply:
> I believe this is just the version it sets up initially.

I'm not sure if you've answered this question before, but do you know why it has to set up an initial version at all? If we can just have it install Bazelisk, we won't need to update this and the builds will run faster.

Reply:
I agree. I honestly haven't looked at this deeply, so it's possible this isn't really doing anything.